site/en/configure/json-trace-profile.md

```diff
@@ -0,0 +1,196 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# JSON Trace Profile
+
+{% include "_buttons.html" %}
+
+The JSON trace profile can be very useful to quickly understand what Bazel spent
+time on during the invocation.
+
+By default, for all build-like commands and query Bazel writes such a profile to
+`command.profile.gz`. You can configure whether a profile is written with the
+[`--generate_json_trace_profile`](/docs/reference/command-line-reference#flag--generate_json_trace_profile)
+flag, and the location it is written to with the
+[`--profile`](/docs/user-manual#profile) flag. Locations ending with `.gz` are
+compressed with GZIP. Use the flag
+[`--experimental_announce_profile_path`](/docs/reference/command-line-reference#flag--experimental_announce_profile_path)
+to print the path to this file to the log.
+
+## Tools 
```

Comment:
nit: remove trailing whitespace

Reply:
Done (checked all files for trailing whitespaces, too.)